### ISSUE TITLE
Inclusive Excellence

### ISSUE THREAD

**Issue by karennapullen** [karennapullen/accessibility#1]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Division of Inclusive Excellence | Sacramento State](https://www.csus.edu/division-inclusive-excellence/)
#### Element path
.q-row-1.col-md-4:nth-child(2) > a[target="_parent"]
#### Snippet
`<a href="office-disability-justice/index.html" target="_parent">
                            Office for Disability Justice
            </a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4 (foreground color: #ffffff, background color: #897f5a, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 119.0.0.0
